build.gradle

```diff
@@ -56,8 +56,6 @@ project.gradle.taskGraph.whenReady { graph ->
   }
 }
 
-tasks.release.dependsOn(bintrayUpload)
-
 
 task('updateDependencies', description: "Rebuilds spinnaker-dependencies.yml based on Spring's platform BOM.") << {
```

Comment:
this would ultimately be better served by a proper gradle task and proper dependency on the spring platform bom, but I'm hoping to just kill this project off entirely so unless I fail at that I'm ok to leave this as is for now